docs/README.md

```diff
@@ -11,6 +11,10 @@ script, along with whatever supporting code you need, and we manage
 accepting student submissions, running your autograder at scale, and
 distributing the results back to students and to you.
 
+!!! note "Updates"
```

Comment:
Wanted to try moving this under the "Overview" section and using an "Admonition" to style this. I think this looks pretty nice, and hopefully eye-catching but not *too* loud?

The options for styles are here: https://sphinx-rtd-theme.readthedocs.io/en/latest/demo/demo.html#admonitions for this message.

Reply:
Ah, THIS is what I was looking for, haha. The blockquote+emoji was my best attempt at replicating this (couldn't find anything in the docs).